Formula/mesa.rb

```diff
@@ -121,10 +154,10 @@ def install
 
     system "./autogen.sh", *args
     system "make"
-    system "make", "-C", "xdemos", "DEMOS_PREFIX=#{prefix}"
+    system "make", "-C", "xdemos", "DEMOS_PREFIX=#{prefix}" if build.without?("only-cpu")
```

Comment:
after changing the option to `without-gpu`, change the conditional to `if build.with? "gpu"`. No parentheses.

Reply:
Ok.